Added example on CRI parsing longlogs (#242)
coralogix/telemetry-shippers: logs/fluentd/k8s-helm/http/README.md

## Coralogix Fluentd Buffer Alert

In order to create an alert on Fluentd buffer in Coralogix, please see [coralogix-alert doc](https://github.com/coralogix/telemetry-shippers/blob/master/logs/fluentd/docs/coralogix-alerts.md)

## Log Logs: containerd / CRI partial logs

> If your application is generating logs longer than 16k you should notice that docker dirver is splitting the log in multiple messages.

Reviewer: ```suggestion
If your application is generating logs longer than 16k, you should note that the Docker driver will split the logs into multiple messages.
```

> To fix this we can use concat to fix this.

Reviewer: ```suggestion
To resolve this, you can use the `concat` filter.
```

> First lets make sure that in the override file, that you use to deploy the helm, has logtag as one of the regex group keys, just like this.

Reviewer: ```suggestion
First ensure that in the override file used to deploy the Helm chart has `logtag` as one of the regex group keys, like the following:
```

```yaml
<pattern>
  format /^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$/
  time_format %Y-%m-%dT%H:%M:%S.%L%z
  keep_time_key true
</pattern>
```
If that is not the case please replace the existing one with this one.

> Then next to the source we will add the following filter that will concat the logs:

Reviewer: ```suggestion
Then, next to the source, you'll need to add the following filter that will concat the logs:
```
